packages/user-interface/src/components/SidenavDivider.tsx

```diff
@@ -0,0 +1,7 @@
+import styles from "./SidenavDivider.module.scss";
+
+const SidenavDivider = () => {
```

Comment:
Dit is iets wat bij de Den Haag Sidenav hoort, dus ik zou verwachten dat dit component ook daar vandaan zou komen: https://nl-design-system.github.io/denhaag/?path=/story/react-navigation-sidenav--default.

Reply:
Yes ik heb nu de spanning uit het component zelf gehaald